Source/SPConnectionController.m

```diff
@@ -1326,6 +1364,35 @@ - (IBAction)exportFavorites:(id)sender
 	 }];
 }
 
+#pragma mark -
+#pragma mark Time Zone changes
+
+- (IBAction)didChangeSelectedTimeZone:(NSPopUpButton *)sender
+{
+	NSMenuItem *selectedItem = [sender selectedItem];
+	switch (selectedItem.tag) {
+		case SPUseServerTimeZoneTag:
+			[self setTimeZoneMode:SPConnectionTimeZoneModeUseServerTZ];
+			[self setTimeZoneIdentifier:@""];
+			break;
+		case SPUseSystemTimeZoneTag:
+			[self setTimeZoneMode:SPConnectionTimeZoneModeUseSystemTZ];
+			[self setTimeZoneIdentifier:@""];
```

Comment:
I think this line is incorrect for setting server to system tz

Reply:
The timezone identifier for this mode is generated when you connect. https://github.com/Sequel-Ace/Sequel-Ace/pull/307/files#diff-892501fd16c85fb7683b43c3139a761bR2389